themes/app/src/js/components/CustomNav.vue

```diff
@@ -1,38 +1,62 @@
 <template>
     <nav class="main-nav">
-        <div class="main-nav__top-menu">
-            <a class="main-nav__logo" href="/"><img/></a>
-            <button v-if="isMobile"
-                class="main-nav__menu-hamburger btn"
-                @click="hamClicked" type="button">
-                &#9776;
-            </button>
-            <div v-if="!isMobile" class="main-nav__right-menu">
-                <slot name="form"></slot>
-                <slot name="button"></slot>
+        <div class="container">
+            <div class="main-nav__top-menu row justify-content-between">
+                <a class="main-nav__logo col-md-6 col-6" href="/"><img/></a>
```

Comment:
Is the `<img/>` element being used here at all? If not, should it be removed?

Reply:
It is being used, the src is set in scss